Cython/Compiler/Pipeline.py

```diff
@@ -136,13 +136,12 @@ def inject_utility_code_stage(module_node):
                 for dep in utilcode.requires:
                     if dep not in added:
                         utility_code_list.append(dep)
-            tree = utilcode.get_tree(cython_scope=context.cython_scope)
-            if tree:
+            if tree := utilcode.get_tree(cython_scope=context.cython_scope):
                 module_node.merge_in(tree.with_compiler_directives(),
                                      tree.scope, stage="utility",
                                      merge_scope=True)
-            elif hasattr(utilcode, 'pxd_scope') and utilcode.pxd_scope:
-                module_node.scope.cimported_modules.append(utilcode.pxd_scope)
+            elif shared_library_scope := utilcode.get_shared_library_scope(cython_scope=context.cython_scope):
```

Comment:
I think this is the first time anyone's tried to get a `:=` operator into Cython. (No comment on if that's good or bad; just an observation)